test/gitea_results_annotation_test.go

```diff
@@ -0,0 +1,65 @@
+//go:build e2e
+// +build e2e
+
+package test
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"testing"
+
+	"github.com/openshift-pipelines/pipelines-as-code/pkg/kubeinteraction"
+	tgitea "github.com/openshift-pipelines/pipelines-as-code/test/pkg/gitea"
+	"github.com/openshift-pipelines/pipelines-as-code/test/pkg/options"
+	"gotest.tools/v3/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	AnnotationPullRequest = "pipelinesascode.tekton.dev/pull-request"
```

Comment:
this is in the keys so we can refer from there
https://github.com/openshift-pipelines/pipelines-as-code/blob/29ebe61af966cba78676b09f7bc9190bd419c95a/pkg/apis/pipelinesascode/keys/keys.go#L40C1-L41C1

Reply:
ah! sure. Thanks for pointing out. 🙃